test/any/Diff-test.js

```diff
@@ -0,0 +1,961 @@
+      expect(chunks.length).toBe(0);
+    });
+
+    it('should return a chunk if getter differs in definition', function() {
+      var firstMap = createMap(`
+        interface Test {
+          getter boolean getByIndex(unsigned long index);
+        };`);
+      var secondMap = createMap(`
+        interface Test {
+          getter any getByIndex(unsigned long index);
+        };`);
+
+      var chunks = differ.diff(firstMap, secondMap);
+      expect(chunks.length).toBe(1);
+      // Expecting the difference to be in the type of the definition.
+      expect(chunks[0].propPath).toBe('.definition.members[getByIndex].member.returnType.name.literal');
+      expect(chunks[0].status).toBe(DiffStatus.VALUES_DIFFER);
+      expect(chunks[0].leftValue).toBe('boolean');
+      expect(chunks[0].rightValue).toBe('any');
+    });
+
+    it('should return a chunk if getter (or setter) is missing from one definition', function() {
+      var firstMap = createMap(`
+        interface Test {
+          getter any getByIndex(unsigned long index);
+          setter void setByIndex(unsigned long index, any value);
+        };`);
+      var secondMap = createMap(`
+        interface Test {
+          getter any getByIndex(unsigned long index);
+          void setByIndex(unsigned long index, any value);
+        };`);
+
+      var chunks = differ.diff(firstMap, secondMap);
+      expect(chunks.length).toBe(1);
+      // Expecting the difference to be in the qualifiers
+      expect(chunks[0].propPath).toBe('.definition.members[setByIndex].member.qualifiers');
+      expect(chunks[0].status).toBe(DiffStatus.NO_MATCH_ON_RIGHT);
+      expect(chunks[0].leftValue).toBeDefined();
+      expect(chunks[0].rightValue).toBeUndefined();
+    });
+
+    it('should return no chunks for overloaded functions rearranged', function() {
+      var firstMap = createMap(`
+        interface Test {
+          void f();
+          void f(double x);
+          void g();
+          void g(DOMString x);
+        };`);
+      var secondMap = createMap(`
+        interface Test {
+          void g(DOMString x);
+          void f(double x);
+          void f();
+          void g();
+        };`);
+
+      var chunks = differ.diff(firstMap, secondMap);
+      expect(chunks.length).toBe(0);
+    });
+
+    it('should return 2 diff fragments for missing static attributes', function() {
+      var firstMap = createMap(`
+        interface Circle {
+          static readonly attribute long triangulationCount;
+          Point triangulate(Circle c1, Circle c2, Circle c3);
+        };`);
+      var secondMap = createMap(`
+        interface Circle {
+          readonly attribute long triangulationCount;
+          static Point triangulate(Circle c1, Circle c2, Circle c3);
+        };`);
+
+      var chunks = differ.diff(firstMap, secondMap);
+      expect(chunks.length).toBe(2);
+      // Expecting the differences to be at member level (since static is a
+      // different type of member).
```

Comment:
static vs non-static definition causes the definition to differ at the member level. This is because static wraps around the member (e.g. static has a field which contains the operation for static functions).